projects/core/components/scroll-controls/scrollbar.directive.ts

```diff
@@ -20,6 +21,7 @@ import {TuiOrientation} from '@taiga-ui/core/enums';
 import {TUI_ELEMENT_REF, TUI_SCROLL_REF} from '@taiga-ui/core/tokens';
 import {fromEvent, interval, merge, Observable} from 'rxjs';
 import {map, switchMap, takeUntil} from 'rxjs/operators';
+import {TuiScrollbarWrapperDirective} from './scrollbar-wrapper.directive';
 
 const MIN_WIDTH = 24;
 
```

Comment:
You can remove the TODO on the line below as it is already implemented :)

Reply:
Removed